vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -0,0 +1,105 @@
+---
+  - name: Label BMO CRDs.
+    shell: 'kubectl label --overwrite crds baremetalhosts.metal3.io {{ item }} --overwrite '
+    with_items:
+       - clusterctl.cluster.x-k8s.io=""
+       - cluster.x-k8s.io/provider="metal3"
+    
+  - name: Define username variable as metal3 for target node
+    set_fact:
+      username: metal3
+
+  - name: Obtain target cluster kubeconfig
+    shell: "kubectl get secrets {{ CLUSTER_NAME }}-kubeconfig -n {{ NAMESPACE }} -o json | jq -r '.data.value'| base64 -d > /tmp/target.yaml"
+    register: target_kubeconfig
+
+  - name: Pivoting test
+    delegate_to: "{{ CLUSTER_APIENDPOINT_IP }}"
+    vars:
+      ansible_user: "{{ username }}"
+      ansible_ssh_private_key_file: "{{ SSH_PRIVATE_KEY }}"
+      ansible_python_interpreter: /usr/bin/python3
+    become: yes
+    become_user: "{{ username }}"
+    block:
+      - name: Create namespace
+        k8s:
+          name: "{{ NAMESPACE }}"
+          kind: Namespace
+          state: present
+          namespace: 
+
+      - name: Clone Baremetal Operator
+        git:
+          repo: 'https://github.com/metal3-io/baremetal-operator.git'
+          dest: "/home/{{ username }}/baremetal-operator"
+            
+      - name: Download kustomize
+        get_url:
+          url: https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh
+          dest: "/home/{{ username }}/baremetal-operator/"
+          mode: '655'
+        register: kustomize_manifest
+
+      # Install Kustomize
+      - name: Install kustomize
+        shell: "bash /home/{{ username }}/baremetal-operator/install_kustomize.sh"
+
+      # Replacing the name of the bridge
+      - name: Replace the POD_CIDR in calico config
+        replace:
+          path: /home/{{ username }}/baremetal-operator/deploy/ironic_ci.env
+          regexp: "provisioning"
+          replace: "{{ PROVISIONING_INTERFACE }}"
+
+      # Install ironic
+      - name: Install kustomize
+        shell: "bash /home/{{ username }}/baremetal-operator/tools/run_local_ironic.sh"
+        environment:
+          CONTAINER_RUNTIME: "docker"
```

Comment:
Is it always docker? Should it rather be based on the value of `CONTAINER_RUNTIME` variable?

Reply:
Done